Amazon.QLDB.Driver/driver/QldbDriver.cs

```diff
@@ -16,33 +16,43 @@ namespace Amazon.QLDB.Driver
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    using System.Threading;
+    using Amazon.QLDBSession;
     using Amazon.Runtime;
+    using Microsoft.Extensions.Logging;
 
     /// <summary>
     /// <para>Represents a factory for accessing a specific ledger within QLDB. This class or
-    /// <see cref="QldbDriver"/> should be the main entry points to any interaction with QLDB.
+    /// <see cref="AsyncQldbDriver"/> should be the main entry points to any interaction with QLDB.</para>
```

Comment:
QldbDriver?

Reply:
"This class or <see cref="AsyncQldbDriver"/> should be..."

I don't see an issue.